### ISSUE TITLE
Set CHUID is much harder with .NET SDK than with yubico-piv-tool

### ISSUE THREAD

**Issue by gnida-rada** [Yubico/Yubico.NET.SDK#4]
With yubico-piv-tool, changing CHID was a matter of running:
_yubico-piv-tool --key=<management key> -a set-chuid_
NOTE: one does not need to supply chuid, yubico-piv-tool just creates random data for us.

with .NET SDK, I don't see high level API to achieve the same result. I need to go through several commands:
_InitializeAuthenticateManagementKeyCommand_
_CompleteAuthenticateManagementKeyCommand_
and, the most painful, 
_PutDataCommand(PivDataTag.Chuid, ...)_
The last one is the most painful because I need to construct encoded CHUID data.

**Comment by GregDomzalski** [comment]
@gnida-rada @nadcraker I just wanted to circle back and make you both aware that we added APIs for creating/managing common PIV objects in 1.3.0. Thanks for your feedback, and I hope this new feature works well for you.

https://docs.yubico.com/yesdk/yubikey-api/Yubico.YubiKey.Piv.Objects.CardholderUniqueId.html
https://docs.yubico.com/yesdk/yubikey-api/Yubico.YubiKey.Piv.Objects.CardCapabilityContainer.html

**Comment by nadcraker** [comment]
@GregDomzalski Yes! I saw the 1.3.0 update and have already been playing with the code. Thanks a lot for this. It makes my job easier. Do you have a roadmap yet for the toolkit? I would be interested to see what plans you have for it in the future.
